apine.schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#", 
+  "$id": "https://github.com/INCF/apine/blob/master/apine.schema.json", 
+  "type": "object", 
+  "additionalProperties": false, 
+  "properties": {
+    "dataset": {
+      "$id": "/properties/dataset", 
+      "minLength": 1,
+      "type": "string", 
+      "title": "dataset", 
+      "description": "The parent" 
+    },
+    "participant": {
+      "$id": "/properties/participant", 
+      "type": "string", 
+      "title": "participant", 
+      "description": "The corresponding participant"
+    }, 
+    "session": {
+      "$id": "/properties/session", 
+      "type": "string", 
+      "title": "session",
+      "description": "The corresponding session"
+    },
+    "modality": {
+      "$id": "/properties/modality", 
+      "type": "string", 
+      "title": "modality", 
+      "description": "The corresponding modality", 
+      "examples": [
```

Comment:
Is it "examples" or  "is_one_of" ? I don't think there are other possibilities and it would be good to list them all there and tell people that these are the only possible values.
Which makes me think we should link to the specific BIDS spec id.

Reply:
Great point! I changed `examples` to `enum` here, forcing it to be one of these. I'm not sure I understand what you mean by linking to the BIDS spec id.